t/admin/routes.t

```diff
@@ -1369,6 +1369,7 @@ passed
 location /t {
     content_by_lua_block {
         local t = require("lib.test_admin").test
+        local core = require("apisix.core")
```

Comment:
why change this test case file?

Reply:
that is a not related thing. Travis report failed sometimes because of this test case.